### PR TITLE
refactor(docs-infra): remove obsolete ViewEngine code path from `ElementsLoader`

### PR DESCRIPTION
**Pull request by gkalpak**
Remove a code path that was only reached in ViewEngine mode and is now (with the removal of ViewEngine) obsolete.

### CODE REVIEW

File: aio/src/app/custom-elements/elements-loader.ts [hunk 1]
```diff
@@ -52,21 +45,8 @@ export class ElementsLoader {
...
-              return elementModuleOrFactory;
-            } else {
-              return this.compiler.compileModuleAsync(elementModuleOrFactory);
-            }
-          })
+        (modulePathLoader() as Promise<Type<WithCustomElementComponent>>)
+          .then(elementModule => this.compiler.compileModuleAsync(elementModule))
           .then(elementModuleFactory => {
             const elementModuleRef = elementModuleFactory.create(this.moduleRef.injector);
             const injector = elementModuleRef.injector;
```
Comment (AndrewKushnir) [inline aio/src/app/custom-elements/elements-loader.ts:52]: Just a drive by comment: I think you could get rid of the `Compiler` symbol here as well by leveraging the https://angular.io/api/core/createNgModuleRef function (you may not need to resolve/compile the module factory).

**Review by petebacondarwin — APPROVED** [review]

File: aio/src/app/custom-elements/elements-loader.ts (hunk 1 above)
Comment (gkalpak) [reply aio/src/app/custom-elements/elements-loader.ts:52]: 🎉

Thank you so much, @AndrewKushnir ❤️
I looked for something like this, but I couldn't find anything.

Comment (AndrewKushnir) [reply aio/src/app/custom-elements/elements-loader.ts:52]: Wlcm :)

**Comment by dylhunn** [comment]
This PR was merged into the repository by commit f7ea5249deb98d691cac64edc4ae38404ee47a06.

**Pull request closed without merging**